src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -147,6 +147,7 @@ var HTMLDOMPropertyConfig = {
     srcDoc: 0,
     srcLang: 0,
     srcSet: 0,
+    srcObject: 0,
```

Comment:
The difference is between:

```js
video.setAttribute('srcObject', stream)
```

and

```js
video.srcObject = stream
```

Since stream is likely an instance of `MediaStream`, setting it as an attribute wouldn't work since it would coerce it to a string

```html
<video srcobject="[object MediaStream]"></video>
```